environment.yml

```diff
@@ -41,6 +41,7 @@ dependencies:
   - libprotobuf=3.21.12
   - pip:
     - git+https://github.com/popsim-consortium/stdpopsim.git
+    - git+https://github.com/popgenmethods/smcpp
```

Comment:
Not sure if/how smcpp was getting installed before, but I got errors it didn't exist until I added this.